### PR TITLE
Detector Variation Fcls For Calibration Production

### PR DESCRIPTION
**Pull request by VCLanNguyen**

### CODE REVIEW

**Review by linyan-w — APPROVED** [review]

**Review by annab101 — APPROVED** [review]

**Comment by VCLanNguyen** [comment]
@bear-is-asleep 
I keep simplemerge data products after reco1 so that we can rerun reco1 for detector variation sample.

**Review by bear-is-asleep — COMMENTED** [review]
I approve not dropping this, it adds ~1-2 MB per event.

File: sbndcode/JobConfigurations/base/reco_drops.fcl [hunk 1]
```diff
@@ -21,8 +21,8 @@ reco_drops: [ @sequence::detsim_drops,
               "drop raw::RawDigits_*_*_*",
               "drop raw::OpDetWaveforms_*_*_*",
               "drop *_cluster3d_*_*", #drop all mlreco output
-              "drop *_simplemerge_*_*", #drop all mlreco output
               "drop *_sedlite_*_*" #drop all mlreco output
+              #"drop *_simplemerge_*_*" #drop all mlreco output
```
Comment (bear-is-asleep) [inline sbndcode/JobConfigurations/base/reco_drops.fcl:25]: I approve not dropping this, it adds ~1-2 MB per event

File: sbndcode/JobConfigurations/base/reco_sbnd.fcl [hunk 2]
```diff
@@ -289,8 +289,8 @@ outputs:
          "drop sim::OpDetBacktrackerRecords_*_*_*",
          #"drop *_linecluster_*_*",
          "drop *_fasthit_*_*",
-         "drop *_cluster3d_*_*", #drop all mlreco output
-         "drop *_simplemerge_*_*" #drop all mlreco output
+         "drop *_cluster3d_*_*" #drop all mlreco output
+         #"drop *_simplemerge_*_*" #drop all mlreco output
```
Comment (bear-is-asleep) [inline sbndcode/JobConfigurations/base/reco_sbnd.fcl:293]: ok

**Review by bear-is-asleep — APPROVED** [review]

**Comment by fjnicolas** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_87_00

**Pull request merged**